<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>100</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reports Management</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logos/bus1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>781</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load_all_driv">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Driver reports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load_all_tick">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ticketsman reports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_loadall">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>All reports</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Search by data</string>
    </property>
    <widget class="QDateEdit" name="data_search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_search_by_data">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>261</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Search by worker</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>firstname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>lastname</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Search by worker</string>
    </property>
    <widget class="QLineEdit" name="line_firstname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>firstname</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_lastname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lastname</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_search_by_name">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="table_reports">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>691</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Worker</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vehicle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Route</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mileage</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vehicle Problems</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Work Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Registred Money</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogout">
    <property name="title">
     <string>Logout</string>
    </property>
    <addaction name="action_back_to_menu"/>
    <addaction name="actionlogout"/>
   </widget>
   <addaction name="menuLogout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogout">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="action_back_to_menu">
   <property name="text">
    <string>Back to menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
